src/app/pages/News/News.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import useFetch from '../../hooks/useFetch';
+import Card from '../../components/Card/Card';
+import Header from '../../components/Header/Header';
+import ToTop from '../../components/Icons/ToTop';
+import Navigation from '../../components/Navigation/Navigation';
+import styles from './News.module.css';
+
+interface NewsMotd {
+  data: {
+    motds: [
+      {
+        id: string;
+        title: string;
+        body: string;
+        image: string;
+      }
+    ];
+  };
+}
+
+interface NewsStw {
+  data: {
+    messages: [
+      {
+        title: string;
+        body: string;
+        image: string;
+      }
+    ];
+  };
+}
+
+export default function News(): JSX.Element {
+  const { data: battleRoyaleData } = useFetch<NewsMotd>(
+    'https://fortnite-api.com/v2/news/br'
+  );
+  const battleRoyaleNews = battleRoyaleData?.data.motds;
```

Comment:
I think this works as long as your API is working, please consider using the loading state from `useFetch`